visualize-admin/visualization-tool
# Refactor/chart config custom color palettes

**This PR:**
- This PR is meant to fully cover the refactoring of the chart config for issue #1868 
- This PR will also include migrations moving foward
- This PR Adds. the Custom Color Picker 


### How to test
1. Go to this [Link](https://visualization-tool-git-refactor-chart-config-custom-514bd6-ixt1.vercel.app)
2. Choose a Chart (ideally Einmalvergütung für Photovoltaikanlagen)
3. Select Vertical Axis Y
4. See new possibility to change color ✅
5. Click on color palette icon, change the color with the new color picker ✅
6. Add a Segmentation, see how here you can select different colors using the new color picker ✅
7. Choose a Combo Chart Type (Dual Axis Line or Column-Line )
8. Select Vertical Axis Y
9. See how you can use custom color picker here as well ✅

---

cc: @bprusinowski , @KerstinFaye , @AnninaWalker , @sosiology
